tests/behat/serve-bootstrap.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+use SilverStripe\CMS\Model\SiteTree;
```

Comment:
I'll implement the below suggestion anyway; Make it an explicit copy, so we won't need this here in any case. :)